integration/internal/container/exec.go

```diff
@@ -0,0 +1,86 @@
+package container
+
+import (
+	"bytes"
+
+	"github.com/docker/docker/api/types"
+	"github.com/docker/docker/client"
+	"github.com/docker/docker/pkg/stdcopy"
+	"golang.org/x/net/context"
+)
+
+// ExecResult represents a result returned from Exec()
+type ExecResult struct {
```

Comment:
Actually; perhaps this type doesn’t have to be exported

Reply:
Done

Reply:
I tried it and it the (way too strict IMO) linter gives:

> 15:55:50 integration/internal/container/exec.go:37:80:warning: exported func Exec returns unexported type container.execResult, which can be annoying to use (golint)

so I had to revert it back to they way it was